Gemfile

```diff
@@ -83,6 +83,7 @@ gem 'action_policy-graphql'
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'lookbook', '>= 2.0.0.rc.1'
```

Comment:
Was there a specific reason for moving the lookbook gem from the development group into this group?

Reply:
I moved it, because I got these errors...

![image](https://github.com/phac-nml/irida-next/assets/325703/a20a4976-613e-4e63-bf3e-5fa872d4d5f5)

Reply:
This branch is the only one that has tests failing without this modification to the Gemfile.  I did some looking around and it appears to be caused in the file `selection_preview.rb`.  Try reverting the `Gemfile` and updating line 3 of `selection_preview.rb` to be:
```
class SelectionPreview < ViewComponent::Preview
```
This is how the other previews are set up.

Reply:
I didn't think I was implementing a ViewComponent, just a stimulus controller. I think that's why I thought I should use Lookbook::Preview instead of ViewComponent::Preview. My implementation isn't a subclass of ViewComponent::Base. But it fixes the error. Please see f6d382edd6ade340811620b7511566a80dd0e8de.